.readthedocs.yaml

```diff
@@ -3,7 +3,7 @@ version: "2"
 build:
   os: ubuntu-22.04
   tools:
-    python: "mambaforge-22.9"
+    python: "mambaforge-latest"
```

Comment:
```suggestion
    python: "mambaforge-23.11"
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/187/commits/8906c9c7cb962fbc3324b167e866b09186f3fa8b